components/match2/commons/match_group_input_util.lua

```diff
@@ -1085,6 +1085,42 @@ function MatchGroupInputUtil.getCharacterName(alias, character)
 	return (assert(alias[character:lower()], 'Invalid character:' .. character))
 end
 
+---@param players {name: string?}[]?
+---@param playerInput string?
+---@param playerLink string?
+---@param options {pagifyPlayerNames: boolean?}?
+---@return integer?
+function MatchGroupInputUtil.findPlayerId(players, playerInput, playerLink, options)
+	if not players then
+		return
+	end
+
+	if not Logic.isEmpty(playerInput) then
```

Comment:
- doesn't make much senese to me, like don't we want the oposite?
- (if it is correct imo use Logic.isNotEmpty)

Reply:
Nah it's a typo